Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py

```diff
@@ -439,669 +1377,758 @@ class Client(BaseClient):
-            score = Common.DBotScore.BAD
-        else:
-            score = Common.DBotScore.NONE
+            system_severity = 3
+        return system_severity
 
-        return Common.DBotScore(
-            indicator=value,
-            indicator_type=type_,
-            integration_name="GIB TI&A",
-            score=score
+    def get_incident_created_time(self) -> str:
+        occured_date_field = INCIDENT_CREATED_DATES_MAPPING.get(
+            self.collection_name, "-"
         )
 
-    if indicator_type == "IP":
-        return Common.IP(ip=value, asn=fields.get("asn"), geo_country=fields.get("geocountry"),
-                         geo_description=fields.get("geolocation"),
-                         dbot_score=calculate_dbot_score(DBotScoreType.IP))
-    elif indicator_type == "Domain":
-        return Common.Domain(domain=value, registrar_name=fields.get("registrarname"),
-                             dbot_score=calculate_dbot_score(DBotScoreType.DOMAIN))
-    elif indicator_type == "File":
-        return Common.File(md5=value, sha1=fields.get("sha1"), sha256=fields.get("sha256"),
-                           name=fields.get("gibfilename"), size=fields.get("size"),
-                           dbot_score=calculate_dbot_score(DBotScoreType.FILE))
-    elif indicator_type == "URL":
-        return Common.URL(url=value, dbot_score=calculate_dbot_score(DBotScoreType.URL))
-    elif indicator_type == "CVE":
-        return Common.CVE(id=value, cvss=fields.get("cvss"), published=fields.get("published"),
-                          modified=fields.get("cvemodified"), description=fields.get("cvedescription"))
-    return None
-
-
-def find_iocs_in_feed(feed: dict, collection_name: str) -> list:
-    """
-    Finds IOCs in the feed and transform them to the appropriate format to ingest them into Demisto.
-
-    :param feed: feed from GIB TI&A.
-    :param collection_name: which collection this feed belongs to.
-    """
-
-    indicators = []
-    indicators_info = MAPPING.get(collection_name, {}).get("indicators", [])
-    for i in indicators_info:
-        main_field = find_element_by_key(feed, i["main_field"])
-        main_field_type = i["main_field_type"]
-        add_fields = []
-        add_fields_list = i.get("add_fields", []) + ["evaluation.severity"]
-        add_fields_types = i.get("add_fields_types", []) + ["severity"]
-        for j in add_fields_list:
-            add_fields.append(find_element_by_key(feed, j))
-        parsed_info = transform_to_command_results(main_field, main_field_type,
-                                                   add_fields, add_fields_types, collection_name)
-        indicators.extend(parsed_info)
-
-    return indicators
-
+        if isinstance(occured_date_field, str):
+            occured_date_field = [occured_date_field]
+
+        assert isinstance(
+            occured_date_field, list
+        ), f"Expected list or string for occured_date_field, got {type(occured_date_field).__name__}"
```

Comment:
It might be better to raise a DemistoException here instead
```
if not isinstance(occured_date_field, list):
    raise DemistoException(f"Expected list or string for occured_date_field, got {type(occured_date_field).__name__}")
```